src/prob/mld.jl

```diff
@@ -28,7 +28,8 @@ function build_mc_mld(pm::_PM.AbstractPowerModel)
     variable_mc_gen_indicator(pm; relax=true)
     variable_mc_gen_power_setpoint_on_off(pm)
 
-    variable_mc_storage_power_mi(pm; relax=true)
+    variable_mc_storage_indicator(pm, relax=true)
+    variable_mc_storage_power_mi_on_off(pm, relax=true)
```

Comment:
As discussed before, we can probably get away with the the non-on/off here.

Reply:
I will make sure there's an Issue to move this functionality to PowerModelsRestoration and simplify the implementation here